src/frontend/src/helpers/Storage/Interface.ts

```diff
@@ -0,0 +1,80 @@
+export {AbstractStorage,
+        File, FileID, FileBrief,
+        Project, ProjectID, ProjectBrief,
+        Settings, defaultSettings}
+
+abstract class AbstractStorage {
```

Comment:
I had the impression that we are moving to replace the sync with dexie's native support. In which case the projectID shouldn't matter. You just need the FileID to update the entry in the file table.